code/datums/observation/moved.dm

```diff
@@ -28,6 +28,11 @@ GLOBAL_DATUM_INIT(moved_event, /decl/observ/moved, new)
 /atom/Entered(atom/movable/am, atom/old_loc)
 	. = ..()
 	GLOB.moved_event.raise_event(am, old_loc, am.loc)
+	SEND_SIGNAL(src, COMSIG_ENTERED, am, old_loc)
+
+/atom/Exited(atom/movable/exitee, atom/new_loc)
+	. = ..()
+	SEND_SIGNAL(src, COMSIG_EXITED, exitee, new_loc)
```

Comment:
We already have this code in `code/datums/observation/entered.dm` and `exited.dm`